docs/source/_static/image_registration.py

```diff
@@ -0,0 +1,35 @@
+import cv2
+import imageio
+import matplotlib.pyplot as plt
+
+import kornia as K
+import kornia.geometry as KG
+
+
+def load_timg(file_name):
+    """Loads the image with OpenCV and converts to torch.Tensor."""
+    assert os.path.isfile(file_name), f"Invalid file {file_name}"
+    # load image with OpenCV
+    img = cv2.imread(file_name, cv2.IMREAD_COLOR)
+    # convert image to torch tensor
+    tensor = K.image_to_tensor(img, None).float() / 255.
+    return K.color.bgr_to_rgb(tensor)
+
+registrator = KG.ImageRegistrator('similarity')
+
+img1 = K.resize(load_timg('/Users/oldufo/datasets/stewart/MR-CT/CT.png'), (400,600))
```

Comment:
We have to provide this images then instead of hardcoding paths

Reply:
we can do, but this code now it's more illustrative than functional. Ideally we should test it too. We can figure out later once we add more examples

Reply:
OK then